src/Taxjar/TaxjarApi.cs

```diff
@@ -147,7 +151,7 @@ protected virtual T SendRequest<T>(string endpoint, object body = null, Method h
         protected virtual async Task<T> SendRequestAsync<T>(string endpoint, object body = null, Method httpMethod = Method.POST) where T : new()
         {
             var request = CreateRequest(endpoint, httpMethod, body);
-            var response = await apiClient.ExecuteTaskAsync<T>(request);
+            var response = await apiClient.ExecuteAsync<T>(request);
```

Comment:
When we use asynchronous code in our library, we should ensure that we use `ConfigureAwait(false);`.

Here's some documentation, but tl;dr - it affects performance and the synchronization context for those on the .NET framework (doesn't affect core).

https://devblogs.microsoft.com/dotnet/configureawait-faq/

Reply:
Will open in a separate PR. 👍